app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if record.errors.any? %>
+  <div id="error-explanation-<%= record.id || record.try(:temporary_id) %>" class="border border-danger rounded py-3 mb-3 bg-white text-danger">
+    <ul class="my-0">
+      <% record.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  </div>
+<% else %>
+  <div id="error-explanation-<%= record.id || record.try(:temporary_id) %>" class="border border-success rounded p-3 mb-3 bg-white text-success">
+    Great job! Inputs are valid.
+  </div>
+<% end %>
```

Comment:
Not sure how I feel about this - This is only for forms that aren't wired into simple form and we only have one form like that now..

Reply:
I feel like this shouldn't be in shared but I'm not totally bothered by it

Reply:
I vote keep and we can fork logic if we need it to support simple_form too at some point.